Sources/Broadcast/BroadcastViewModifier.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+extension Warp {
+    public struct BroadcastViewModifier: ViewModifier {
```

Comment:
Struct is public but initializer/constructor is not accessible from outside.

Reply:
It is working in the DemoApp so users should be fine. Again copy from Toast so if we want to fix something we can do both on a later branch.